packages/react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -384,6 +384,14 @@ export interface ComboBoxProps<ItemType>
    * Provide the text that is displayed when the control is in warning state
    */
   warnText?: ReactNode;
+
+  /**
+   * Props to be passed directly to the underlying <input> element.
+   * Useful for customizing input behavior, such as setting maxLength, placeholder and other native input attributes.
+   *
+   * Note: These props are merged with internal input props. Custom values will override defaults when applicable.
```

Comment:
```suggestion
   * Specify native input attributes to place on the `<input>`, like maxLength.
   * These are passed to downshift's getInputProps() and will override the 
   * internal input props.
   * https://github.com/downshift-js/downshift?tab=readme-ov-file#getinputprops
```

This same comment can be mirrored to the other component as well as the proptypes

Reply:
Changes are adopted